test/realm_test.dart

```diff
@@ -1843,6 +1843,18 @@ Future<void> main([List<String>? args]) async {
     expect(results.length, 1);
     receivePort.close();
   });
+
+  test('Device info', () {
```

Comment:
I think this test is better to be located at `app_test.dart`.  And probably `deviceName` and `deviceVersion` could be properties of AppConfigurationInternal.

Reply:
This is not related to app at all. The native implementation is exposed as realm_dart_xxx as well.